README.md

```diff
@@ -11,29 +11,27 @@
 </h1>
 
 <h3 align="center">
-🖍️ 🛡 🚀
+🛠 💪 💉
 </h3>
 
 <h3 align="center">
-Modular Redux is possible.
+Maintaining large Redux applications with ease.
```

Comment:
it is soooo looooong

Reply:
i think it describes the purpose of this collection better :)